keyboards/keebio/quefrency/keymaps/GeorgePetri/readme.md

```diff
@@ -0,0 +1,48 @@
+# George Petri's Quefrency 65 layout
+
+```
+make keebio/quefrency:GeorgePetri
+```
+
+Based on the default querty layout with minor tweaks. 
```

Comment:
You need to have double spaces at the end of each line, or they won't properly break.   If you view the file on github, you'll see what I mean.